experiments/tauscan.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+"""
+Tauscan experiment
+
+Copywrite SuperDARN 2020
+
+Keith Kotyk
+
+"""
+import os
+import sys
+import copy
+import numpy as np
+
+BOREALISPATH = os.environ['BOREALISPATH']
+sys.path.append(BOREALISPATH)
+
+from experiment_prototype.experiment_prototype import ExperimentPrototype
+import experiments.superdarn_common_fields as scf
+
+class Tauscan(ExperimentPrototype):
+    """A 11-pulse sequence that consists of a single pulse pulse followed by a back to back 5-pulse
+    Farley sequence. The analysis produces a 12-pulse ACF with no missing lags."""
+    def __init__(self):
+        cpid = 503
+
+        super(Tauscan, self).__init__(cpid)
```

Comment:
really like comment_string=FullScanStepMode.__doc__ from the other mode, can you add that here as well